### PR TITLE
implement-forgot-password-functionality

### PR DESCRIPTION
**Pull request by Eben-Success**
## Summary

This pull request implements the edit post functionality as outlined in the project requirements. This functionality allows users to edit either Content or Tag. 

## Impact

- Users will be able to edit their posts.
- Backend API has been updated to handle edit requests.

### CODE REVIEW

File: backend/Application/Contracts/PostRequests/EditPostRequest.cs
Comment (georgeey123) [inline backend/Application/Contracts/PostRequests/EditPostRequest.cs:1]: remove post_Id field from obeject. unnecessary

**Review by georgeey123 — CHANGES_REQUESTED** [review]
remove post_id field from object. Seemsn redundant

**Comment by Eben-Success** [comment]
I've made the changes.

**Review by georgeey123 — COMMENTED** [review]
LGTM

**Review by georgeey123 — APPROVED** [review]

**Pull request merged**